src/chigame/games/models.py

```diff
@@ -6,13 +6,95 @@
 
 class Game(models.Model):
     """
-    A game (Chess, Checkers, etc.)
+    A game like Chess, Checkers, etc.
     """
 
-    name = models.TextField()
+    # Basic information
+    name = models.CharField(max_length=255)
     description = models.TextField()
+    image = models.URLField(default="/static/images/no_picture_available.png")
+    year_published = models.PositiveIntegerField(null=True)
+
+    # Gameplay information
+    rules = models.TextField(null=True)
+
     min_players = models.PositiveIntegerField()
     max_players = models.PositiveIntegerField()
+    suggested_age = models.PositiveSmallIntegerField(
+        null=True
+    )  # Minimum recommendable age. For example, 8+ would be 8.
+
+    expected_playtime = models.PositiveIntegerField(null=True)  # In Minutes
+    min_playtime = models.PositiveIntegerField(null=True)
+    max_playtime = models.PositiveIntegerField(null=True)
+
+    complexity = models.PositiveSmallIntegerField(null=True)  # 1-5, 1 being the easiest
+    category = models.ManyToManyField("Category", related_name="games")
+    mechanics = models.ManyToManyField("Mechanic", related_name="games")
+
+    # BGG information
+    BGG_id = models.PositiveIntegerField(null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Person(models.Model):
+    """
+    A person associated with a game, such as a designer or artist.
+    """
+
+    DESIGNER = 1
+    ARTIST = 2
+
+    ROLE_CHOICES = (
+        (DESIGNER, "Designer"),
+        (ARTIST, "Artist"),
```

Comment:
Given that there are only currently two roles listed, how will you decide what other roles to add?
Things like gameplay tester, director, translator, etc... 

Also can this model reasonably handle the situation where one person has multiple roles on the same work? Such as a solo dev doing all of the parts themselves, or a duo team coding and writing everything?

Reply:
These two roles are listed on board game geek which makes them easy first choices. We were thinking that other roles could just be added as needed. While still have not decided what other ChiGame specific people information needs to be tracked, this person model seems ideal for tracking simple relationships (e.g. developer of a game) as opposed to something more complex (staff members who created or edited a game). 

In regards to the second question, I think it is fine to have one person with multiple roles have different entries for each role. See Pascal Trigaux in [this](https://boardgamegeek.com/xmlapi2/thing?id=20&stats=1) BGG API response.

Reply:
Sounds good! Thank you for the clarification and citation.